source/_posts/2023-09-01-php-core-roundup-16.md

```diff
@@ -0,0 +1,374 @@
+
+---
+
+Welcome back to [PHP Core Roundup](/blog/tag/roundup/) series! This is post #16, where we highlight and celebrate the improvements made to PHP during the month past by the PHP development team, members of the PHP Foundation, and more.
+
+> The PHP Foundation is a collective of PHP contributors, veterans, and companies that collaborate to ensure the long-term sustainability of the PHP programming language. The foundation currently supports six contributors focused on PHP's maintenance, debt reduction, and feature development. These contributors work closely with others on coding, documentation, and discussions.
+
+> Started over a year ago, the PHP Core Roundup series offers a summary of the latest developments, discussions, and news about PHP Core, contributed by both PHP Foundation members and other participants. This post is the thirteenth in the PHP Core Roundup series.
+
+{% include "newsletter.html" %}
+
+## PHP 8.3 Branching Out
+
+PHP 8.3 has reached its [feature-freeze](/blog/2023/08/01/php-core-roundup-15/#php-8.3-feature-freeze), and [release managers](/blog/2023/05/02/php-core-roundup-12/#php-8.3-release-managers-elected) branched out the PHP-8.3 branch on August 31.
+
+Now that PHP 8.3 is in a separate branch, the `master` branch will be the development source for PHP 8.4. Bug fixes and other improvements will be cherry-picked for PHP 8.3 (and older branches) as appropriate, but new features that are made to the master branch will not be merged to the PHP 8.3 branch.
+
+Tools that build PHP based on the Git branches will also see a new branch, and the builds from the `master` branch will be named “PHP 8.4” for the first time.
+
+## Releases
+
+The PHP development team released five new versions in August 2023:
+
+**[PHP 8.2.10](https://www.php.net/archive/2023.php#2023-08-31-2)** and **[PHP 8.1.23](https://www.php.net/archive/2023.php#2023-08-31-3)**
+
+These releases include several bug fixes and improvements, notably in areas such as CLI, Date, Core, DOM, FFI, Hash, MySQLnd, Opcache, PCNTL, SPL, and Standard.
+
+**[PHP 8.2.9](https://www.php.net/archive/2023.php#2023-08-16-1)**, **[PHP 8.1.22](https://www.php.net/archive/2023.php#2023-08-03-1)**, and **[PHP 8.0.30](https://www.php.net/archive/2023.php#2023-08-04-1)**
+
+All three include security fixes: [GHSA-3qrf-m4j2-pcrr](https://github.com/php/php-src/security/advisories/GHSA-3qrf-m4j2-pcrr) and [GHSA-jqcx-ccgc-xwhv](https://github.com/php/php-src/security/advisories/GHSA-jqcx-ccgc-xwhv).
+
+PHP 8.2.9 and PHP 8.1.22 additionally include several bug fixes and improvements, notably in areas such as Build, CLI, Core, Curl, Date, DOM, Fileinfo, FTP, GD, Intl, MBString, Opcache, PCNTL, PDO, PDO SQLite, Phar, PHPDBG, Session, Standard, Streams, SQLite3, and XMLReader.
+
+Please note, that Windows binaries for PHP 8.2.9 are not synchronized and don't contain a fix for [GH-11854](https://github.com/php/php-src/issues/11854). Windows binaries for PHP 8.0.30 are missing so far.
```

Comment:
Builds are already being prepared, I guess this paragraph will be irrelevant after a while
```suggestion
```

Reply:
Ah, thanks. Removed in 9b9fcbe